<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentClass</class>
 <widget class="QDialog" name="AddStudentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddStudent</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>添加学生</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName">
       <property name="placeholderText">
        <string>请输入姓名</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leId">
       <property name="placeholderText">
        <string>请输入学号（最少8位）</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMale">
       <property name="text">
        <string>男</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbFemale">
       <property name="text">
        <string>女</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexBtnGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbAge">
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>院系：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbColleage">
       <item>
        <property name="text">
         <string>计算机学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>通信学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>法学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>英语系</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>兴趣：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbInteresting">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>游泳</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>骑车</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>篮球</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>音乐</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,3,3">
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AddStudent.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="sexBtnGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
